### PR TITLE
refactored docker images replication to idf

### PR DESCRIPTION
**Pull request by srinivasreddych**
refactored docker images replication to idf with unit-tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by srinivasreddych** [comment]
```
---------- coverage: platform darwin, python 3.9.1-final-0 -----------
Name                            Stmts   Miss  Cover
---------------------------------------------------
helmparser/__init__.py              0      0   100%
helmparser/arguments.py             9      0   100%
helmparser/helm/__init__.py         0      0   100%
helmparser/helm/commands.py        25      0   100%
helmparser/logging.py              10      0   100%
helmparser/parser/__init__.py       0      0   100%
helmparser/parser/parser.py        61      0   100%
helmparser/utils/utils.py           8      0   100%
---------------------------------------------------
TOTAL                             113      0   100%

Required test coverage of 80% reached. Total coverage: 100.00%
```

File: manifests/local/deployment.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ groups:
     path: manifests/local/networking-modules.yaml
   - name: storage
     path: manifests/local/storage-modules.yaml
+  - name: replication
+    path: manifests/local/replicator-modules.yaml
```
Comment (dgraeber) [inline manifests/local/deployment.yaml:13]: Is replication now required for EKS?  If this is a local deployment (as opposed to local-isolated), what is the need for the image replication?

File: manifests/local/replicator-modules.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: replication
+path: modules/replication/dockerimage-replication/
+dataFiles:
+  - filePath: data/eks_dockerimage-replication/versions/1.25.yaml
+  - filePath: data/eks_dockerimage-replication/versions/default.yaml
+parameters:
```
Comment (dgraeber) [inline manifests/local/replicator-modules.yaml:6]: Why is replication needed here?

File: manifests/local/deployment.yaml [hunk 3]
```diff
@@ -9,6 +9,8 @@ groups:
     path: manifests/local/networking-modules.yaml
   - name: storage
     path: manifests/local/storage-modules.yaml
+  - name: replication
```
Comment (dgraeber) [inline manifests/local/deployment.yaml]: Why replication module?

**Review by dgraeber — APPROVED** [review]

**Pull request merged**